### PR TITLE
Bug 1850849: disable kibana metrics to address CVEs

### PR DESCRIPTION
**Pull request by ewolinetz**
Addresses:
* https://bugzilla.redhat.com/show_bug.cgi?id=1850849
* https://bugzilla.redhat.com/show_bug.cgi?id=1850850

### CODE REVIEW

File: hack/logging-dump.sh [hunk 1]
```diff
@@ -194,17 +195,34 @@ check_collector() {
...
+  pods=$(oc -n $namespace get pods -l name=cluster-logging-operator -o jsonpath={.items[*].metadata.name})
+  for pod in $pods
+  do
+    get_env $pod $clo_folder $namespace "Dockerfile-*operator*"
+    get_pod_logs $pod $clo_folder $namespace
+  done
+  oc -n $namespace get deployment cluster-logging-operator -o yaml > $clo_folder/deployment
+  csv_name="$(oc -n $namespace get csv -o name | grep 'cluster-logging-operator')"
+  oc -n $namespace get "${csv_name}" -o yaml > "${clo_folder}/csv"
+}
```
Comment (periklis) [inline hack/logging-dump.sh:211]: This is really really cool! Can we expand this on the catalog and olm operator logs? Maybe also get the describe of the subscription and the install plan too. They are super useful if the CRDs are broken or similar bugs raise up.

File: hack/logging-dump.sh [hunk 2]
```diff
@@ -251,9 +273,14 @@ check_kibana() {
...
     check_kibana_connectivity $pod
+    get_kibana_cr
   done
 }
 
+get_elasticsearch_cr() {
+  oc get -n ${NAMESPACE} elasticsearch elasticsearch -o yaml > $es_folder>cr
+}
+
 get_elasticsearch_status() {
```
Comment (periklis) [inline hack/logging-dump.sh:284]: For ES6 we need to expand this on the index management pod logs too.

**Comment by ewolinetz** [comment]
oops, accidentally added in an unrelated file :laughing:

File: hack/logging-dump.sh (hunk 1 above)
Comment (ewolinetz) [reply hack/logging-dump.sh:211]: whoops, this file shouldn't be part of this pr!

**Pull request merged**